decidim-comments/app/resolvers/decidim/comments/vote_comment_resolver.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module Decidim
+  module Comments
+    # A GraphQL resolver to handle `upVote` and `downVote` mutations
+    # It creates a vote for a comment by the current user.
+    class VoteCommentResolver
+      def initialize(options = { weight: 1 })
+        @weight = options[:weight]
+      end
+
+      def call(obj, _args, ctx)
+        Decidim::Comments::VoteComment.call(obj, ctx[:current_user], weight: @weight) do
+          on(:ok) do |comment|
```

Comment:
I'm not handling invalid responses by graphql yet because I'm not sure how to 😕 . Let me do this in a separate PR because it's a bit tricky and I need to do it for comments as well. I need to handle this on the frontend part and the current design may not support it yet. For example, if I create a comment I can update the UI optimistically but what happen if the comment wasn't created? We can add a retry button or just delete the comment, etc.